pkg/registries/registries_test.go

```diff
@@ -408,16 +757,195 @@ func TestEditRegistriesConfig(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "imageTagMirrorSet",
+			itmsRules: []*apicfgv1.ImageTagMirrorSet{
+				{
+					Spec: apicfgv1.ImageTagMirrorSetSpec{
+						ImageTagMirrors: []apicfgv1.ImageTagMirrors{ // other.com is neither insecure nor blocked
+							{Source: "insecure.com/ns-i1", Mirrors: []apicfgv1.ImageMirror{"blocked.com/ns-b1", "other.com/ns-o1"}},
+							{Source: "blocked.com/ns-b/ns2-b", Mirrors: []apicfgv1.ImageMirror{"other.com/ns-o2", "insecure.com/ns-i2"}},
+							{Source: "other.com/ns-o3", Mirrors: []apicfgv1.ImageMirror{"insecure.com/ns-i2", "blocked.com/ns-b/ns3-b", "foo.insecure-example.com/bar"}},
+						},
```

Comment:
This test does not set anything to insecure nor blocked, so using the `insecure.com`/`blocked.com` domains is a bit unexpected.

What is being tested here? Just a smoke-test of correctly creating tag entries? Two entries would be enough for that.